qualtran/testing.py

```diff
@@ -204,3 +217,86 @@ def execute_notebook(name: str):
         nb = nbformat.read(f, as_version=4)
     ep = ExecutePreprocessor(timeout=600, kernel_name="python3")
     ep.preprocess(nb)
+
+
+class BloqCheckResult(Enum):
```

Comment:
Do you think it's time to create a folder for `testing/` and have separate files for the separate logics? I suspect this file will continue to grow and become hard to manage at some point, but we can punt it for later as well. Good to keep this at the back of the mind as we continue to add more checks.

Reply:
Sorry, what's the separate logic? I think this one file is pretty cohesive for now (and only 300 lines)

Reply:
a bunch of functions to test consistency of composite bloqs, which is in some sense different from consistency checks for a Bloq in general. I imagine going forward this would evolve into a `testing/` directory like cirq and each consistency check would have it's own file. But we don't need to do that right now